command/pr_review.go

```diff
@@ -106,15 +109,15 @@ func prReview(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("did not understand desired review action: %w", err)
 	}
 
-	out := colorableOut(cmd)
+	stderr := cmd.ErrOrStderr()
```

Comment:
I think that this should be made "colorable" so Windows can properly output colors on stderr

Reply:
good catch